services/migration_service/migration_config.py

```diff
@@ -1,9 +1,2 @@
 from services.utils import DBConfiguration
-
-db_conf = DBConfiguration()
-
```

Comment:
@oavdeev I have removed the exposure of host, port, user, and password by exposing a single db_conf object that has a URL encoded connection string that can be given to goose.